prowler/providers/github/services/repository/repository_immutable_releases_enabled/repository_immutable_releases_enabled.metadata.json

```
@@ -0,0 +1,31 @@
+{
+  "Provider": "github",
+  "CheckID": "repository_immutable_releases_enabled",
+  "CheckTitle": "Ensure repository immutable releases are enabled",
```

```suggestion
  "CheckTitle": "Repository enforces immutable releases",
```

Following new [metadata guidelines](https://docs.prowler.com/developer-guide/check-metadata-guidelines#common-mistakes-to-avoid).